### PR TITLE
Added auto-detection of input file measurement

### LINKED ISSUE #7: Auto-detect input file measurement
Use input file's classification/metadata (if available) to determine measurement so the user does not have to select it. This will allow this gear to run automatically in many cases. 

Could have measurement options as such:
1. Auto-Detect (default)
2. Functional
3. T1w
4. T2w

### PR DESCRIPTION
**Pull request by jenreiter**
This fixes #7 as well as #6 

I have tested the gear on Functional, T1w and T2w input images here. All inputs can be seen here: https://gears.dev.flywheel.io/#/dashboard/project/59b8296bc4fdd5001e1e8184/session/59d55dfa2c4d1c001c9472b5?tab=data

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -84,35 +95,65 @@ else
     else
         measurement=`cat $MANIFEST_FILE | jq -r '.config.measurement.default'`
     fi
+
+    ## Determine measurement
+    # Check if autodetect, otherwise use what was read in from config.json/manifest.json
+    if [[ $measurement = 'auto-detect' ]]; then
+        echo "Auto-detecting input file measurement"
+        # Get measurement from config
+        measurement=`cat $CONFIG_FILE | jq '.inputs.nifti.object.measurements[0]'`
```
Comment (lmperry) [inline run]: I believe you can use `jq -r` and you would not need the next two lines.

**Review by lmperry — APPROVED** [review]
LGTM. Up to you if you want to simplify based on comment.

**Comment by lmperry** [comment]
@jenreiter: I'm considering the case where we would want to automatically run this gear over any nifti file in a session...  
To accomplish that, we need to default to auto-detect and exit if the classification is not one of the supported types. I think we should make that change in the logic. 

Does that make sense?

File: run (hunk 1 above)
Comment (jenreiter) [reply run]: Implemented this change.

**Comment by jenreiter** [comment]
@lmperry I understand.

To confirm, you are saying that I should check if the auto-detected classification is 'functional', 'anatomy_t1w' or 'anatomy_t2w', if it is not one of those options, I should raise an error. 

What should I do for the case if there is no classification? Currently, I assign to 'functional'. Should I change this to also raise an error?

**Comment by lmperry** [comment]
Yes, it should do nothing. It's arguable if it should be an error, or a 'nothing to do, exiting (0)'.

**Pull request merged**